### PR TITLE
Update Jib versions post-3.1.2 release

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -332,7 +332,7 @@ In Maven, you can use the `maven-resources-plugin` to copy files to your extra d
   ...
   <plugin>
     <artifact>maven-resources-plugin</artifact>
-    <version>3.1.1</version>
+    <version>3.2.0</version>
```
Comment (chanseokoh) [inline docs/faq.md:335]: I realized there doesn't exist 3.1.1 for `maven-resources-plugin`. I think it was a wrong update during the post release process in the past.

Comment (mpeddada1) [reply docs/faq.md:335]: That is a great catch!

File: jib-maven-plugin/src/test/resources/maven/projects/simple/pom-jar-containerization.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
-        <version>3.1.1</version>
+        <version>3.2.0</version>
```
Comment (mpeddada1) [inline jib-maven-plugin/src/test/resources/maven/projects/simple/pom-jar-containerization.xml:40]: `maven-jar-plugin`'s latest version is 3.2.0. I'm assuming we're intentionally updating it here?

**Review by mpeddada1 — APPROVED** [review]

File: jib-maven-plugin/src/test/resources/maven/projects/simple/pom-jar-containerization.xml (hunk 2 above)
Comment (chanseokoh) [reply jib-maven-plugin/src/test/resources/maven/projects/simple/pom-jar-containerization.xml:40]: Yeah, `maven-jar-plugin` does have 3.1.1, but I think we can always use the latest.

**Pull request merged**